data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton_storage.json

```diff
@@ -0,0 +1,323 @@
+[
+  {
+    "id": "exo_rifle_storage",
```

Comment:
Most of the items in this file could use copy-from to save lines and also to make it easier to maintain these in the future.

Reply:
Changing this back reminded me why I changed it in the first place, the base items cause an error because they use  "torso_hanging_back" which should only use the BELTED layer.